src/ypd/templates/CS320-ProjectMainPage.html

```diff
@@ -1,51 +1,66 @@
-        <link href="../static/TeamProjectStyleSheet.css" rel="stylesheet" type="text/css">
-    </head>
-    
-    <body>
-        <div class="mainTitle">
-            <div class="logo">
-                <img src="../static/Logo.png" alt="Logo" width="30" height="35">
-            </div>
-            <center>YCP Project Database</center>
 
-            <div class="dropDown">
-                <div class="menuTab"><center><div class="bar"></div><div class="bar"></div><div class="bar"></div></center></div>
-                <div class="dropDown-Content">
-                    <a href="{{ url_for('IndexView:get') }}">Main Page</a>
-                    <a href="{{ url_for('SubmissionView:post') }}">Post A Project</a>
-                    <!-- <a href="{{ "url_for('UserView:login')" }}">Login/Sign Up</a> -->
-                </div>
-            </div>
+<head>
+    <title>YCP Project Database</title>
+    <link rel="icon" href="Logo.png" type="image/png">
+    <link href="https://fonts.googleapis.com/css?family=Alfa+Slab+One&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Rokkitt&display=swap" rel="stylesheet">
+    <link href="../static/TeamProjectStyleSheet.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+    <div class="mainTitle">
+        <div class="logo">
+            <img src="../static/Logo.png" alt="Logo" width="30" height="35">
         </div>
-        
-        <div class="searchBar">  
-            <form action="/search">
-                <input type="text" name="search_text" size="12" placeholder="Search.." value="{{search_text}}"/>
-                <input type="submit" value="Search">
-            </form>
+        <center>YCP Project Database</center>
+
+        <div class="dropDown">
+            <div class="menuTab">
+                <center>
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                </center>
+            </div>
+            <div class="dropDown-Content">
+                <a href="{{ url_for('IndexView:get') }}">Main Page</a>
+                <!-- <a href="{{ " url_for( 'SubmissionView:submission') " }}">Post A Project</a> -->
+                <a href="{{ url_for( 'UserView:login') }}">Login/Sign Up</a>
+            </div>
         </div>
-       
-        
-        <div class="mainBody">
-            {% for item in catalog.projects %}
-                <div class="postBox">
-                    <div class="logoIn"><div class="logoInPad"><!--<img src="{{ item.logo }}" alt="Logo" width="25" height="30">--></div></div>
-                        <!-- <a href="selectedproject?id={{item.id}}{% if item is provided %}&isProvided=True{% endif %}"> -->
-                        <a href={{url_for('SelectedProjectView:get', id=item.id, is_provided=(item is provided))}}>
-                            <div class="postText">
```

Comment:
This is supposed to be a link to view a project from the list of projects. It needs to stay in the final version of this file.